app/(main)/register/page.test.tsx

```diff
@@ -170,4 +195,11 @@ describe('Register', () => {
 
     expect(darkModeSection).toHaveClass('dark:bg-gradient-to-b');
   });
+
+  test('Register submission button is set to loading state when clicked', async () => {
```

Comment:
This test needs to render the page itself. Right now, it's rendering a mock component. If any changes occurred in register/page.tsx, this current test would not catch it.

Reply:
I changed the test to check the actual component